graviti/manager/common.py

```diff
@@ -6,12 +6,20 @@
 """Common tools."""
 
 import warnings
+from enum import Enum
 from typing import TYPE_CHECKING
 
 if TYPE_CHECKING:
     from graviti.manager.commit import Commit
 
 
+class DatasetStatus(Enum):
+    """An enumeration type representing the current status of the dataset."""
+
+    CURRENT_COMMIT = object()
+    CURRENT_BRANCH = object()
```

Comment:
normally we use `auto()` to generate enum values automatically.
https://docs.python.org/zh-cn/3/library/enum.html#enum.auto